omero/sysadmins/config.txt

```diff
@@ -1180,13 +1180,23 @@ Default: `pixelDataEventLogQueue`
 omero.pixeldata.max_plane_height
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
+With :property:`omero.pixeldata.max_plane_width` specifies
+the plane size cutoff above which a pixel pyramid should
+either be read directly from the file format or generated
+by the pixeldata service.
```

Comment:
I do like the direction of your suggested rephrasing but am also happy not to block this PR going in as-is.